### PR TITLE
Update: Cleanup all jquery extends

### PR DESCRIPTION
**Pull request by balajimanoharan**
Update: Cleanup all jquery extends

### CODE REVIEW

**Comment by cythrawll** [comment]
Do we need to do ember polyfill explicitly? does babel polyfills already include `Object.extend`?

**Comment by balajimanoharan** [comment]
> Do we need to do ember polyfill explicitly? does babel polyfills already include Object.extend?

Do you mean this? 
```
import { assign } from '@ember/polyfills';
```
That was just the new way to import the assign thats already a part of `Ember`.
https://emberjs.com/api/ember/3.2/functions/@ember%2Fpolyfills/assign
I didn't have to bring in a polyfill.

File: packages/reports/addon/routes/reports/report.js [hunk 1]
```diff
@@ -200,7 +201,7 @@ export default Ember.Route.extend({
       let metadata = get(this, 'currentModel.visualization.metadata');
 
       set(this.currentModel, 'visualization.metadata',
-        Ember.$.extend(true, {}, metadata, metadataUpdates)
+        merge({}, metadata, metadataUpdates)
```
Comment (Xuefeng-Zhu) [inline packages/reports/addon/routes/reports/report.js]: why not use merge from Ember?

**Comment by cythrawll** [comment]
> Do you mean this?

> `import { assign } from '@ember/polyfills';`
> That was just the new way to import the assign thats already a part of Ember.
> https://emberjs.com/api/ember/3.2/functions/@ember%2Fpolyfills/assign

Yeah how is that different than the native `Object.assign`? Maybe it works with ember objects?

> I didn't have to bring in a polyfill.

I suspect the polyfill is already there.

File: packages/reports/addon/routes/reports/report.js (hunk 1 above)
Comment (balajimanoharan) [reply packages/reports/addon/routes/reports/report.js]: `Ember.merge` works only for 2 objects

**Comment by balajimanoharan** [comment]
> Yeah how is that different than the native Object.assign? Maybe it works with ember objects?

Its the same thing. Looks like some guy called @jkusa did this.. https://github.com/emberjs/ember.js/blob/c1f4d5b80279cba9424b83505dc28d65cada7572/packages/%40ember/polyfills/lib/assign.ts#L52

**Pull request merged**